xiilib/databases.py

```diff
@@ -90,6 +156,9 @@ def get_uris(database_requirers: typing.Dict[str, DatabaseRequires]) -> typing.D
 
         env_name = f"{interface_name.upper()}_DB_CONNECT_STRING"
 
+        if interface_name == "redis":
+            endpoint = data["endpoints"].split(",")[0]
+            db_uris[env_name] = f"{interface_name}://{endpoint}"
```

Comment:
When scaled up redis is deployed with Sentinel, we should leverage sentinel in such cases:
https://redis.readthedocs.io/en/stable/examples/connection_examples.html#Connecting-to-a-Sentinel-instance

Reply:
The issue with sentinel is that it lacks a standardized URI format. Additionally, we do not support high availability solutions for other databases (except MongoDB) because there is no URI format for high availability for them either. We likely need to consider a solution for the database's HA configuration.